adapters/blue/params_test.go

```diff
@@ -0,0 +1,50 @@
+package blue
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+// This file actually intends to test static/bidder-params/blue.json TODO: MUST BE CREATED
```

Comment:
Please delete the TODO part of this comment or the entire comment.

Reply:
done